kratos/containers/csr_matrix.h

```diff
@@ -151,8 +151,8 @@ class CsrMatrix
     //move assignement operator
     CsrMatrix& operator=(CsrMatrix&& rOtherMatrix)
     {
+        mIsOwnerOfData=rOtherMatrix.mIsOwnerOfData;
```

Comment:
@roigcarlo this was also missing (please comment)

Reply:
Yep, :+1: